cheddar/cheddar-server/src/main/java/com/clicktravel/cheddar/server/runtime/config/RuntimeConfiguration.java

```diff
@@ -61,10 +61,11 @@
 
     public static boolean isLocalEnvironment(final Environment environment) {
         for (final String profile : environment.getActiveProfiles()) {
-            if (profile.equalsIgnoreCase(LOCAL_PROFILE)) {
+            if (profile.equalsIgnoreCase(LOCAL_PROFILE) || profile.equalsIgnoreCase(TEST_PROFILE)
```

Comment:
OK lets leave local as it was b4 i changed it. I'll add a new method for isDeployedEnvironment to check for (ci | uat | prod)